Public/Src/Engine/Processes/ReportedFileAccess.cs

```diff
@@ -218,20 +218,7 @@ public string ShortDescribe(PathTable pathTable)
         /// <summary>
         /// Determines whether the current violation is a write violation
         /// </summary>
-        public bool IsWriteViolation
-        {
-            get {
-                switch (RequestedAccess)
-                {
-                    case RequestedAccess.All:
-                    case RequestedAccess.Write:
-                    case RequestedAccess.ReadWrite:
-                        return true;
-                    default:
-                        return false;
-                }
-            }
-        }
+        public bool IsWriteViolation => (RequestedAccess & RequestedAccess.Write) != 0;

```

Comment:
Since we are moving to NetCore and away from Net472 we can start using HasFlag. Since it doesn't have boxing cost in dotnetcore..